SpecFlowProjectMars1/StepDefinitions/LanguageSkills.cs

```diff
@@ -0,0 +1,143 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using SpecFlowProjectMars1.Pages;
+using SpecFlowProjectMars1.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+using SeleniumExtras.WaitHelpers;
+
+namespace SpecFlowProjectMars1.StepDefinitions
+{
+    [Binding]
+    public class LanguageSkills:CommonDriver
+    {
+        private IWebDriver driver;
+        private WebDriverWait wait;
+        Language languageobj = new Language();
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors